dcrjson/walletsvrcmds.go

```diff
@@ -524,6 +524,24 @@ func NewSignRawTransactionCmd(hexEncodedTx string, inputs *[]RawTxInput, privKey
 	}
 }
 
+// VerifySeedCmd defines the verifyseed JSON-RPC command.
+type VerifySeedCmd struct {
+	Seed    string  `json:"seed"`
```

Comment:
Commands should not have the json field name overrides because they are positional.  Take a look at all of the other `type XCmd struct` definitions and notice that none of them do so.